### ISSUE TITLE
docke build fails: Couldn't find package "@qshift/plugin-maven-backend@workspace:^" required by "backend@0.0.0" on the "npm" registry.

### ISSUE THREAD

**Issue by cmoulliard** [q-shift/backstage-playground#115]
## Issue

The job fails to build the container image - see: https://github.com/q-shift/backstage-playground/actions/runs/9257646218/job/25466220154

```
 #13 26.24 Error: Couldn't find package "@qshift/plugin-maven-backend@workspace:^" required by "backend@0.0.0" on the "npm" registry.
#13 26.24     at MessageError.ExtendableBuiltin (/opt/yarn-v1.22.19/lib/cli.js:721:66)
#13 26.24     at new MessageError (/opt/yarn-v1.22.19/lib/cli.js:750:123)
#13 26.24     at PackageRequest.<anonymous> (/opt/yarn-v1.22.19/lib/cli.js:36590:17)
#13 26.24     at Generator.throw (<anonymous>)
#13 26.24     at step (/opt/yarn-v1.22.19/lib/cli.js:310:30)
#13 26.24     at /opt/yarn-v1.22.19/lib/cli.js:323:13
#13 26.24     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
#13 26.24 Error: Couldn't find any versions for "@qshift/plugin-quarkus-backend" that matches "workspace:^"
#13 26.24     at MessageError.ExtendableBuiltin (/opt/yarn-v1.22.19/lib/cli.js:721:66)
#13 26.24     at new MessageError (/opt/yarn-v1.22.19/lib/cli.js:750:123)
#13 26.24     at Function.<anonymous> (/opt/yarn-v1.22.19/lib/cli.js:50351:13)
#13 26.24     at Generator.next (<anonymous>)
#13 26.24     at step (/opt/yarn-v1.22.19/lib/cli.js:310:30)
#13 26.24     at /opt/yarn-v1.22.19/lib/cli.js:321:13
#13 26.24     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
#13 ERROR: process "/bin/sh -c yarn install --frozen-lockfile --production --network-timeout 300000" did not complete successfully: exit code: 1
```

The issue can be reproduced locally:
```
yarn tsc
yarn build:all
yarn build-image -t quay.io/ch007m/backstage-qshift:latest --push
```

**Comment by cmoulliard** [comment]
Job to build the kubernetes image succeeded: https://github.com/q-shift/backstage-playground/actions/runs/9258158105/job/25467749384 with the PR merged

**Comment by cmoulliard** [comment]
Fixed with PRs: #120 #116 #117
